### ISSUE TITLE
[Bug]: Auckland Council, New Zealand Nolonger working

### ISSUE THREAD

**Issue by shanekuz** [mampfes/hacs_waste_collection_schedule#2046]
### I Have A Problem With:

A specific source

### What's Your Problem

The auckland council integration seems to have stopped, I have checked the path and this and the area value still look to be the same.  Im happy to try and collect logs if i can get a steer as to which logs would help

### Logs

```Shell
no relevant logs
```

### Relevant Configuration

```YAML
Config

waste_collection_schedule:
  sources:
    - name: aucklandcouncil_govt_nz
      args:
        area_number: **********

Sensors

  - platform: waste_collection_schedule
    name: rubbish
    value_template: '{{value.daysTo}}'
    types:
      - rubbish

  - platform: waste_collection_schedule
    name: rubbish_day
    types:
      - rubbish

  - platform: waste_collection_schedule
    name: recycle
    value_template: '{{value.daysTo}}'
    types:
      - recycle 

  - platform: waste_collection_schedule
    name: recycle_day
    types:
      - recycle
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
Duplicate of #1962

**Comment by 5ila5** [comment]
Fix is part of the newly released version 1.48.0

**Comment by shanekuz** [comment]
thank you all working for me now.
